DESCRIPTION

```diff
@@ -14,5 +14,14 @@ LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.0.2
 Additional_repositories: https://mrc-ide.github.io/drat
+Imports:
+    hintr,
+    jsonlite,
+    pkgapi,
+    traduire
```

Comment:
I'd be inclined to put minimum version numbers against pkgapi and traduire as those packages are evolving in response to this one